SplunkRumWorkspace/SplunkRum/SplunkRumTests/SpanTests.swift

```diff
@@ -0,0 +1,41 @@
+//
+/*
+Copyright 2021 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import XCTest
+@testable import SplunkOtel
+
+class SpanTests: XCTestCase {
+    func testEventSpan() throws {
+        // Forces RUM to reinitialze for testing
+        SplunkRum.initialized = false
+        _ = SplunkRum.initialize(beaconUrl: "http://127.0.0.1:8989/",
+                                 rumAuth: "FAKE_RUM_AUTH",
+                                 options: SplunkRumOptions(allowInsecureBeacon: true,
+                                                           debug: true,
+                                                           globalAttributes: [:],
+                                                           environment: nil,
+                                                           ignoreURLs: nil,
+                                                           sessionSamplingRatio: 0.5)
+                                 )
+
+        let dictionary: NSDictionary = [
+                        "attribute1": "hello",
+                        "attribute2": "world!"
+        ]
+        SplunkRum.reportEvent(name: "testEvent", attributes: dictionary)
```

Comment:
@seemk Should this be documented in the following page?

- https://docs.splunk.com/Observability/gdi/get-data-in/rum/ios/manual-rum-ios-instrumentation.html#nav-Manually-instrument-applications

Reply:
When I was looking at how to do this on iOS vs Android I used this page.

https://docs.splunk.com/Observability/rum/RUM-custom-events.html

In the Android example it was the first code snippet that this is adding a function for.

Reply:
@bgustafson Which snippets would you use to replace the current iOS example?

Reply:
I would add one.  This is an event that has no time component to it just that something happened.    So this would be the same code just not trying to time something.

```swift
        SplunkRum.reportEvent(name: "calculateTax", attributes: [
            "numClaims": "claims.count",
            "workflow.name": "<your_workflow>!"
        ])
```